packages/type-safe-api/src/project/type-safe-api-project.ts

```diff
@@ -256,16 +256,16 @@ export class TypeSafeApiProject extends Project {
     const libraries = [...new Set(options.library?.libraries ?? [])];
 
     const libraryDir = "libraries";
-    const libraryDirRelativeToParent = parentMonorepo
+    const libraryDirRelativeToParent = nxWorkspace
       ? path.join(options.outdir!, libraryDir)
       : libraryDir;
 
     // Declare the generated runtime projects
     const generatedLibraryProjects = generateLibraryProjects(libraries, {
-      parent: parentMonorepo ?? this,
+      parent: nxWorkspace ? this.parent! : this,
       parentPackageName: this.name,
       generatedCodeDir: libraryDirRelativeToParent,
-      isWithinMonorepo: !!parentMonorepo,
+      isWithinMonorepo: !!this.parent,
```

Comment:
This should be `!!nxWorkspace` :)

Reply:
Will need to update this here and for `generateInfraProject` below as this tells the generated typescript projects whether or not to run the `yarn/pnpm/npm link` commands.

Thinking about it though, I wonder if it actually needs to be `isWithinTsMonorepo` as I suppose with a Python based monorepo we don't have an overarching yarn/pnpm/npm workspace so the packages would still need to be linked in order to see one another?

Reply:
Updated in latest revision